src/elements/action-group/action-group.visual.spec.ts

```diff
@@ -0,0 +1,168 @@
+import { html, nothing } from 'lit';
+
+import {
+  describeEach,
+  describeViewports,
+  visualDiffDefault,
+  visualRegressionFixture,
+} from '../core/testing/private.js';
+
+import './action-group.js';
+import '../button/button.js';
+import '../button/secondary-button.js';
+import '../link/block-link.js';
+
+describe(`sbb-action-group`, () => {
+  let root: HTMLElement;
+
+  const horizontalCases = {
+    orientation: ['horizontal'],
+    states: [
+      { elements: 3, alignGroup: 'start' },
```

Comment:
as this is a single list of entries without combinations, I recommend using a normal for() and not the describeEach(). with that, you also will not have the "default" suffix everywhere.